Automattic/wp-calypso
# A4A: Build Migrations V2 page layout skeleton.

This PR aims to create the layout skeleton for. the new Migrations page. We will achieve this bytingSection component currently utilized on the Marketplace hosting page.

<img width="1724" alt="Screenshot 2024-10-29 at 3 00 51 PM" src="https://github.com/user-attachments/assets/6a50a973-4bbf-4415-a9f8-8ec9231b4181">
<img width="1726" alt="Screenshot 2024-10-29 at 3 01 05 PM" src="https://github.com/user-attachments/assets/5e8ac566-bc93-4f49-8345-f03a030805ff">
<img width="1728" alt="Screenshot 2024-10-29 at 3 01 14 PM" src="https://github.com/user-attachments/assets/8654e610-6f03-4996-991e-2c4abf64db37">

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1374

## Proposed Changes

* As both the Marketplace hosting page and the new Migrations page share the same design language, we will repurpose the HostingSection component for the new page. However, we need to refactor it to create a more universally shared component and rename it to `PageSection`. This will require a few refactoring on existing files that reference it.
* We will create the layout skeleton using the `PageSection`. **For now, we will utilize any available backgrounds from the Marketplace and later replace them with the final design's background.**

## Why are these changes being made?

* This will set the foundational work for the new migrations page.

## Testing Instructions

* Use the A4A live link and go to the `/migrations` page.
* Confirm that you see the layout skeleton with headers and descriptions for each section.
* **Finally, test for regression in the Marketplace hosting page and ensure nothing has been broken with the refactoring.**

## Pre-merge Checklist
